config/locales/views/mailer/hi.yml

```diff
@@ -0,0 +1,39 @@
+en:
+  mailer:
+    welcome_heading: "सर्किटवर्स में आपका स्वागत है, %{name}"
```

Comment:
Can this name be converted to Hindi? It’s showing I’m English even in Hindi translations